opentech/apply/dashboard/tables.py

```diff
@@ -13,3 +18,28 @@ class Meta:
 
     def render_user(self, value):
         return value.get_full_name()
+
+
+def get_used_rounds(request):
+    return Round.objects.filter(submissions__isnull=False).distinct()
+
+
+def get_used_funds(request):
+    # Use page to pick up on both Labs and Funds
+    return Page.objects.filter(applicationsubmission__isnull=False).distinct()
+
+
+class Select2ModelMultipleChoiceFilter(filters.ModelMultipleChoiceFilter):
+    def __init__(self, *args, **kwargs):
+        label = kwargs.get('label')
+        kwargs.setdefault('widget', Select2MultiCheckboxesWidget(attrs={'data-placeholder': label}))
+        super().__init__(*args, **kwargs)
+
+
+class SubmissionFilter(filters.FilterSet):
+    round = Select2ModelMultipleChoiceFilter(queryset=get_used_rounds, label="Rounds")
+    page = Select2ModelMultipleChoiceFilter(queryset=get_used_funds, label='Funds')
```

Comment:
I see the reasoning behind using `page`, but `fund` makes a lot more sense especially when it comes to query parameters

Reply:
missing lead filter